packages/apps/dashboard/server/src/modules/networks/networks.service.ts

```diff
@@ -0,0 +1,90 @@
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import { ChainId, NETWORKS, StatisticsClient } from '@human-protocol/sdk';
+
+import {
+  EnvironmentConfigService,
+  MINIMUM_ESCROWS_COUNT,
+  MINIMUM_HMT_TRANSFERS,
+} from '../../common/config/env-config.service';
+import { AVAILABLE_NETWORKS_CACHE_KEY } from 'src/common/config/redis-config.service';
+import { Cache, CACHE_MANAGER } from '@nestjs/cache-manager';
+import { MainnetsId } from 'src/common/utils/constants';
+
+@Injectable()
+export class NetworksService {
+  private readonly logger = new Logger(NetworksService.name);
+  constructor(
+    @Inject(CACHE_MANAGER) private cacheManager: Cache,
+    private readonly envConfigService: EnvironmentConfigService,
+  ) {}
+
+  public async getOperatingNetworks(): Promise<ChainId[]> {
+    const cachedNetworks = await this.cacheManager.get<ChainId[]>(
+      AVAILABLE_NETWORKS_CACHE_KEY,
```

Comment:
```suggestion
      OPERATING_NETWORKS_CACHE_KEY,
```
And probably cache-key itself as well